netcon/descent3onlineclient/CMakeLists.txt

```diff
@@ -27,6 +27,11 @@ target_link_libraries(Descent3_Online_TCP_IP PRIVATE
 target_include_directories(Descent3_Online_TCP_IP PRIVATE ${SDL3_INCLUDE_DIRS})
 if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
   set_target_properties(Descent3_Online_TCP_IP PROPERTIES SUFFIX ".dylib")
+  if(DEFINED CODESIGN_IDENTITY AND NOT "${CODESIGN_IDENTITY}" STREQUAL "")
```

Comment:
We could make that couple lines a CMake function or macro that takes the target name as argument

Reply:
You can define a function like this in the top level CMakelists:
```cmake
set(CODESIGN_IDENTITY "" CACHE STRING "Sets the macOS code signing identity. If set to something besides the empty string, then the dynamic libraries put into the hog files will be signed using this identity.")
function (macos_sign target)
  message ("target: ${target}")
  if(DEFINED CODESIGN_IDENTITY AND NOT "${CODESIGN_IDENTITY}" STREQUAL "")
  message(STATUS "Code signing ${target}")
  add_custom_command(TARGET ${target} POST_BUILD
         COMMAND codesign --verbose --sign "${CODESIGN_IDENTITY}" --force --timestamp --deep -o runtime $<TARGET_FILE:${target}>)
  endif()  
endfunction()
```

and use it as `macos_sign("Parallax_Online")` for instance

Reply:
Thanks for the help!  Fixed.